alidns/_demo/demo.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	al "github.com/libdns/alidns"
+	l "github.com/libdns/libdns"
+)
+
+func main() {
+	provider := al.Provider{
+		AccKeyID:     "LTAI4G3UTA4x2XRm8HrgGJ63",       //Modify your AccessKeyId here
```

Comment:
Hopefully this it not your token. If so, you sould delete it from alicoud. Git never forgets :-D
For secrets you sould always use "os.Getenv"

Reply:
Thanks,I'll use 'os.Getenv' instead of hard code,and remove this key from alicloud.